README.md

```diff
@@ -4,9 +4,12 @@ In the spirit of [analog-inspection](https://github.com/amiaopensource/analog-in
 
 ### Guidelines and Articles
 
+* **Howard Besser** (2001), [Longevity of Electronic Art
+](http://besser.tsoa.nyu.edu/howard/Papers/elect-art-longevity.html)
 * **Pip Laurensen** (2006), [Authenticity, Change and Loss in the Conservation of Time-Based Media Installations](http://www.tate.org.uk/research/publications/tate-papers/06/authenticity-change-and-loss-conservation-of-time-based-media-installations)
 * **Electronic Arts Intermix** (c.2007), [EAI Online Resource Guide for Exhibiting, Collecting & Preserving Media Art](http://www.eai.org/resourceguide/home.html)
 * **Smithsonian** (May 2010), [Collaborations in Conserving Time-Based Art: A Summary of Discussion Group Sessions of a Colloquium ](https://www.si.edu/content/tbma/documents/Time-Based-Art_Report.pdf)
+* **Dragan Espenschied _et al_** (September 2013), [Large-Scale Curation and Presentation of CD-ROM Art](http://purl.pt/24107/1/iPres2013_PDF/Large-Scale%20Curation%20and%20Presentation%20of%20CD-ROM%20Art.pdf)
```

Comment:
Sorry for being dot-picking: does `et al.` not have a period in English? It is the Latin abbreviation of _et alii_ (masculine), _et aliae_ (feminine) or _et alia_ (neuter).